src/chrome/content/rules/3g2upl4pq6kufc4m.onion.xml

```diff
@@ -0,0 +1,15 @@
+<!--
+	Other DuckDuckGo rulesets:
+		- DuckDuckGo.xml
```

Comment:
Please update the reference list in `DuckDuckGo.xml`

Reply:
Done.